api/src/main/java/org/openmrs/Order.java

```diff
@@ -363,6 +363,38 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated before or on the current date
+	 * @since 2.0
+	 * @see #isActivated(java.util.Date)
```

Comment:
@djazayeri can I get your opinion on this topic. I think @wluyima and I disagree here. he wants a test for case `isActive(null)` which leads checkDate to default to `new Date()`. This hasnt been tested in any method in this class (isActive(), isActive(null), isStarted() and isStarted(null) are not tested). Not saying this is an argument for not doing it but I just dont know how. I would be happy to do so but need more instruction.